contract-address/CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ The format is based on [Keep a Changelog].
 
 ## [Unreleased]
 
+# [0.7.0] - 2021-06-30
```

Comment:
let's remove the date for now, I'd like to get #552 in before we do a release and it's unlikely to be today
here and below
```suggestion
```

Reply:
> I'd like to get #552 in before we do a release

That PR constrains `parity-scale-codec` to a pre-release, which could cause some issues resolving when mixed with other crates with a dependency on `parity-scale-codec`. So we should do a pre-release for that as recommended here: https://doc.rust-lang.org/cargo/reference/resolver.html#pre-releases

For `scale-info`, I did a standalone [`0.8.0-rc`](https://github.com/paritytech/scale-info/releases/) which contained only contained the `parity-scale-codec` pre-release requirement.